crates/symbolicator/src/services/download/mod.rs

```diff
@@ -131,16 +149,15 @@ impl DownloadService {
         });
 
         match result.await {
-            Ok(DownloadStatus::NotFound) => {
-                log::debug!(
-                    "Did not fetch debug file from {:?}: {:?}",
-                    source,
-                    DownloadStatus::NotFound
-                );
-                Ok(DownloadStatus::NotFound)
-            }
             Ok(status) => {
-                log::debug!("Fetched debug file from {:?}: {:?}", source, status);
+                match status {
+                    DownloadStatus::Completed => {
+                        log::debug!("Fetched debug file from {:?}: {:?}", source, status);
```

Comment:
no need to do that in this PR, but the debug logs here print *all* of the source information, including private S3 keys, etc to the console, so that is waaay to detailed.

Reply:
i'll create a follow-up PR that tries to address this in a simple way 👍